go.mod

```diff
@@ -4,13 +4,17 @@ go 1.20
 
 require (
 	github.com/ClickHouse/clickhouse-go/v2 v2.17.1
+	github.com/charmbracelet/lipgloss v0.10.0
 	github.com/go-sql-driver/mysql v1.8.0
 	github.com/jackc/pgx/v5 v5.5.5
 	github.com/mfridman/interpolate v0.0.2
 	github.com/microsoft/go-mssqldb v1.7.0
+	github.com/peterbourgon/ff/v4 v4.0.0-alpha.4
 	github.com/sethvargo/go-retry v0.2.4
+	github.com/stretchr/testify v1.9.0
```

Comment:
I've given up trying to fight this, the rest of the community tends to favor this as the defacto assertion library. Instead of maintaining our own ./internal/check we should just use this.